packages/web-components/fast-element/src/debug.ts

```diff
@@ -21,6 +21,7 @@ const debugMessages = {
     [1206 /* directCallToHTMLTagNotAllowed */]: "Calling html`` as a normal function invalidates the security guarantees provided by FAST.",
     [1207 /* onlySetTemplatePolicyOnce */]: "The DOM Policy for an HTML template can only be set once.",
     [1208 /* cannotSetTemplatePolicyAfterCompilation */]: "The DOM Policy cannot be set after a template is compiled.",
+    [1209 /* blockedByDOMPolicy */]: "'${aspectName}' on '${tagName}' is blocked by the current DOMPolicy.",
```

Comment:
Is there a way to look up these messages by the code? I have seen these get thrown sometimes, but it is just the code that shows in the web console.

Reply:
You need to include the debug module to see the messages. You can import it at application startup. You just don't want to ship with that in there. So, if your build has debug symbols you can use to remove code, that's how you would want to add it.